app/helpers/permission-to-remove-contributor.js

```diff
@@ -8,24 +8,25 @@ import { helper } from '@ember/component/helper';
 /**
  * permissionToRemoveContributor helper.  Checks to see if user has proper permissions
  * to remove contributor.  The user must be an admin and cannot remove herself.
- * The project cannot be a registration.
  *
  * @class permissionToRemoveContributor
  * @param {Object} contributor Contributor you wish to remove.
  * @param {Object} currentUser Current logged in user.
  * @param {Boolean} isAdmin Whether current user is a preprint admin
- * @param {Object} node The preprint itself.
+ * @param {Boolean} editMode Is the preprint in editMode?
  * @return {Boolean} Does current user have permission to remove this particular contributor?
  */
 export function permissionToRemoveContributor(params/* , hash */) {
-    const [contributor, currentUser, isAdmin, node] = params;
+    const [contributor, currentUser, isAdmin, editMode] = params;
     const currentUserId = currentUser.get('currentUserId') || currentUser.get('id');
     const removeSelf = contributor.get('userId') === currentUserId;
-    let isRegistration = null;
-    if (node) {
-        isRegistration = node.get('registration');
+
+    if (editMode) {
+        return (removeSelf || isAdmin);
```

Comment:
Read/write contributors now have permission to remove themselves (this is the only view they can modify contributorship. They previously could modify on the node).  They can only remove themselves; they cannot update their own permissions or remove others.